doc/langref.html.in

```diff
@@ -4199,11 +4199,73 @@ pub fn print(self: *Writer, arg0: []const u8, arg1: i32) !void {
       {#header_close#}
       {#header_close#}
 
+      {#header_open|Multithreading#}
+      <p>
+      The {#link|Zig Standard Library#} contains an abstraction of OS threads as {#syntax#}std.Thread{#endsyntax#}, as well
+      as a number of concurrency primitives under that namespace.
+      </p>
+
+      <p>
+      Zig currently inherits LLVM's memory model, but <a href="https://github.com/ziglang/zig/issues/6396">may
+      define its own one</a> (most likely compatible with LLVM). Until then,
+      <a href="https://llvm.org/docs/LangRef.html#memory-model-for-concurrent-operations">LLVM's documentation</a> is
+      the ground truth.
```

Comment:
I think that lines 4040~4212 are a concise way of explaining the status quo while providing further references 👍 

I'd say that everything below line 4212 is excessive for this PR.
- The text as is attempts to give an introductory overview of motivation for employing a memory consistency model without rigorously defining any particular one. However, I believe the language reference is not the correct place to learn about memory models in general.
- Attempting to rigorously define the LLVM memory model surely would also be outside the scope of Zig's langref.

So, I'd argue that the best thing Zig can do right now is exactly captured in lines 4040~4212 of this PR: explain the status quo, reference LLVM, and note that things might change.

Reply:
The point of a reference is to describe the language accurately, without saying only "go read the formal specification". Why should this principle not apply to the section on multithreading?